### PR TITLE
Add autowrap recipe

### PR DESCRIPTION
**Pull request by aseyboldt**

### CODE REVIEW

File: recipes/autowrap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - autowrap.data_files
+
+  commands:
+    - autowrap --version
+
+about:
+  home: https://github.com/uweschmitt/autowrap
+  license: BSD-3-Clause
+  license_family: BSD
+  # autowrap does not include its license file in the pypi release.
```
Comment (ericdill) [inline recipes/autowrap/meta.yaml]: Might be worth noting that its license file now included in the MANIFEST.in and so

Comment (ericdill) [inline recipes/autowrap/meta.yaml]: ...grr

And so will be included in the next release

File: recipes/autowrap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  run:
+    - python
+    - setuptools
+    - cython ==0.21
+
+test:
+  imports:
+    - autowrap
+    - autowrap.data_files
+
```
Comment (ericdill) [inline recipes/autowrap/meta.yaml]: nit: delete this blank line

File: recipes/autowrap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipes/autowrap/meta.yaml]: Please add [`entry_points`]( http://conda.pydata.org/docs/building/meta-yaml.html#python-entry-points ) like so.

```yaml
  entry_points:
    - autowrap = autowrap.Main:main
```

File: recipes/autowrap/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  build:
```
Comment (jakirkham) [inline recipes/autowrap/meta.yaml]: Please add `toolchain`.

File: recipes/autowrap/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - python
+    - setuptools
+    - cython ==0.21
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/autowrap/meta.yaml]: Please drop `setuptools` after adding the entry point above.

File: recipes/autowrap/meta.yaml (hunk 1 above)
Comment (aseyboldt) [inline recipes/autowrap/meta.yaml]: yes

File: recipes/autowrap/meta.yaml (hunk 2 above)
Comment (aseyboldt) [inline recipes/autowrap/meta.yaml]: done

File: recipes/autowrap/meta.yaml (hunk 3 above)
Comment (aseyboldt) [inline recipes/autowrap/meta.yaml]: done

File: recipes/autowrap/meta.yaml (hunk 4 above)
Comment (aseyboldt) [inline recipes/autowrap/meta.yaml]: done

**Comment by jakirkham** [comment]
Thanks @aseyboldt and thanks @ericdill for the review.

**Pull request merged**